values.yaml

```diff
@@ -471,6 +472,9 @@ rabbitmq-ha:
   # Make sure to also change the rabbitmqMemoryHighWatermark following the formula:
   # rabbitmqMemoryHighWatermark = 0.4 * resources.limits.memory
   resources: {}
+  image:
+    # Image pull policy. Change to "IfNotPresent" when switching to stable images
+    pullPolicy: IfNotPresent
```

Comment:
```suggestion
```
The pull policy is already `IfNotPresent` for the upstream chart. No reason to hardcode it here. Also the registry for rabbitmq-ha chart is outside of Docker Hub